infra/gcp/terraform/k8s-infra-oci-proxy-prod/variables.tf

```
@@ -23,6 +23,11 @@ variable "project_id" {
 variable "digest" {
   type = string
 }
+
+variable "verbosity" {
+  type = string
+}
```

Probably want to default the value 

```suggestion
variable "verbosity" {
  type = string
  default = "0"
}
```